src/game.cpp

```diff
@@ -5687,9 +5687,6 @@ float game::natural_light_level( const int zlev ) const
     for( const auto &e : events ) {
         // EVENT_DIM slowly dims the natural sky level, then relights it.
         if( e.type == EVENT_DIM ) {
-            if( zlev < 0 ) {
```

Comment:
Why? This is an artifact effect that dims the sky temporally, why would it have an effect underground?